Sources/MapboxCoreNavigation/NavigationSettings.swift

```diff
@@ -88,16 +102,20 @@ public class NavigationSettings {
      fall back to the `NavigationSettings.directions` by default.
        - tileStoreConfiguration: Options for configuring how map and navigation tiles are stored on the device. See
      `TileStoreConfiguration` for more details.
+       - navigationRouterType: Configures the type of routing to be used by various SDK objects when providing route calculations.
```

Comment:
This is very vague. What else can we say about the implications of choosing one type versus another?